devito/operator.py

```diff
@@ -280,6 +283,7 @@ def cfunction(self):
         if self._cfunction is None:
             self._cfunction = getattr(self._lib, self.name)
             argtypes = [c_int if isinstance(v, ScalarArgument) else
+                        c_float if isinstance(v, ConstantArgument) else
```

Comment:
Yes, will change. This is also what is breaking dse/dle probalby.
All the `required` in `create_elemental_function` are casted as `ScalarArguments` and `m` may en up in it.
I'll try to remove the hard-coded types.